src/main/java/org/swisspush/redisques/util/RedisquesConfiguration.java

```diff
@@ -42,6 +42,7 @@ public class RedisquesConfiguration {
     private final int memoryUsageLimitPercent;
     private final int memoryUsageCheckIntervalSec;
     private static final int DEFAULT_CHECK_INTERVAL = 60; // 60s
+    private static final int DEFAULT_PROCESSOR_TIMEOUT = 240000; // 240S
```

Comment:
Suggestion:
```patch
-   240000; // 240S
+  240_000;
```
If not, then I suggest to at least use a *small* s for seconds ;)